exchange/exchange.go

```diff
@@ -798,6 +813,24 @@ func applyCategoryMapping(ctx context.Context, requestExt *openrtb_ext.ExtReques
 	return res, seatBids, rejections, nil
 }
 
+func removeBidByName(seatBid *pbsOrtbSeatBid, bidID string) {
```

Comment:
Should this be `removeBidByID` (and the associated test name)?

Reply:
Renamed